manifests/profile/network.pp

```diff
@@ -8,8 +8,16 @@
     ensure => stopped,
     enable => false,
   }
-  exec { 'iptables -F':
-    path => '/sbin',
+
+  #Flush existing rules and save blank ruleset on centos 6
```

Comment:
Centos 7 uses firewalld to manage the firewall and that code was working.  Also this change is only for the student VMs since the other VMs need to run docker which uses iptables.